Program.cs

```diff
@@ -1,10 +1,13 @@
-﻿namespace Training
-{
-    internal class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿// Fibonacci series
+// Given an input eg:n=10, print fibonacci series for f(10)
+
+int firstNumber = 0; int secondNumber = 1;

```

Comment:
Read the previous comment you can declare the variable of same type in a single line.